protocol-correctness/lib/functions/list-elements-are-of-type.k

```diff
@@ -0,0 +1,19 @@
+require "protocol-correctness/lib/language/type-reflection.k"
+
+module LIST-ELEMENTS-ARE-OF-TYPE-SYNTAX
+  imports BASE-TYPES-SYNTAX
+  imports BOOL
+  imports TYPE-REFLECTION-SYNTAX
+
+  syntax Bool ::= listElementsAreOfType(ExpressionList, ReflectionType)  [function, functional]
+endmodule
+
+module LIST-ELEMENTS-ARE-OF-TYPE
+  imports LIST-ELEMENTS-ARE-OF-TYPE-SYNTAX
+
+  rule listElementsAreOfType([.], _:ReflectionType) => true
+  rule listElementsAreOfType([E:Expression, Es:ExpressionCSV], T:ReflectionType)
```

Comment:
why this rule is marked as `simplification`?

Reply:
Most likely it's a copy-paste error. I'll try to fix it.

Reply:
Fixed.